api/api.go

```diff
@@ -420,12 +431,32 @@ type CloudBackupScheduleInfo struct {
 	// MaxBackups are the maximum number of backups retained
 	// in cloud.Older backups are deleted
 	MaxBackups uint
+	// GroupID indicates the group of volumes for this cloudbackup schedule
+	GroupID string
+	// Labels indicates a volume group for this cloudsnap schedule
+	Labels map[string]string
 }
 
 type CloudBackupSchedCreateRequest struct {
 	CloudBackupScheduleInfo
 }
 
+type CloudBackupGroupSchedCreateRequest struct {
+	// GroupID indicates the group of volumes for which cloudbackup schedule is
+	// being created
+	GroupID string
+	// Labels indicates a volume group for which this group cloudsnap schedule is
+	// being created. If this is provided GroupId is not needed and vice-versa.
+	Labels map[string]string
```

Comment:
We should document here what happens when both are provided.

Reply:
When this is moved to api.proto use `oneof` instead, and it will be clear.

Reply:
Yes. I will add. A volume list that matches both criteria will be used.